bin/test

```diff
@@ -6,6 +6,9 @@ go get -t ./...
 
 rm -f coverage.txt
 
+unset AWS_ACCESS_KEY_ID
+unset AWS_SECRET_ACCESS_KEY
```

Comment:
If these aren't unset, then they are present in the test suite environment. In order to test rack installs when no credentials are found, we need there to not be any credentials in the host environment.